R/oc_forward.R

```diff
@@ -1,26 +1,29 @@
 #' Forward geocoding
 #'
-#' Forward geocoding, from placename to latitude and longitude tuple(s).
+#' Forward geocoding from a character vector of placenames to latitude and
+#' longitude tuple(s).
 #'
 # nolint start - link longer than 80 chars
 #' @param placename A character vector with the placename(s) to be geocoded.
-#'   Required. If the placename(s) is/are address(es), see
+#'
+#'   If the placenames are addresses, see
 #'   \href{https://github.com/OpenCageData/opencagedata-misc-docs/blob/master/query-formatting.md}{OpenCage's
-#'    instructions} on how to format addresses for forward geocoding best.
+#'    instructions} on how to format addresses for best forward geocoding
+#'    results.
 # nolint end
 #' @param return A character vector of length one indicating the return value of
 #'   the function, either a list of tibbles (\code{df_list}, the default), a
 #'   JSON list (\code{json_list}), a GeoJSON list (\code{geojson_list}), or the
 #'   URL with which the API would be called (\code{url_only}).
-#' @param key Your OpenCage API key as a character vector of length one.
-#'   Required. By default, \code{\link{oc_key}} will attempt to retrieve the key
-#'   from the environment variable \code{OPENCAGE_KEY}.
+#' @param key Your OpenCage API key as a character vector of length one. By
+#'   default, \code{\link{oc_key}} will attempt to retrieve the key the
```

Comment:
Typo: "from" missing 
```suggestion
#'   default, \code{\link{oc_key}} will attempt to retrieve the key from the
```